lib/centralisation/class/module.dart

```diff
@@ -0,0 +1,67 @@
+import "dart:convert";
+
+class Module {
+  Module({
+    required this.name,
+    required this.description,
+    required this.icon,
+    required this.url,
+    this.liked,
+  });
+  late final String name;
+  late final String description;
+  late final String icon;
+  late final String url;
+  late final bool? liked;
+
+  Module.fromJson(Map<String, dynamic> json) {
+    name = utf8.decode(latin1.encode(json['name']), allowMalformed: true);
+    description = utf8.decode(latin1.encode(json['description']), allowMalformed: true);
```

Comment:
Do you really need to encode and decode, every other classes of other modules works without it ?